src/index.ts

```diff
@@ -1,25 +1,21 @@
 import { assert } from './utils'
 import { createConfig } from './config'
 import { RyuCho } from './ryu-cho'
+import core from '@actions/core'
 
-assert(!!process.env.ACCESS_TOKEN, '`accessToken` is required.')
-assert(!!process.env.USER_NAME, '`userName` is required.')
-assert(!!process.env.EMAIL, '`email` is required.')
-assert(!!process.env.UPSTREAM_REPO, '`upstreamRepo` is required.')
-assert(!!process.env.HEAD_REPO, '`headRepo` is required.')
-assert(!!process.env.TRACK_FROM, '`trackFrom` is required.')
+assert(typeof core !== 'undefined', `core is undefined, which probably means you're not running in a GitHub Action`)
```

Comment:
Does this assertion mean that we can no longer test the whole logic in our local environments?
If so, how we can test its behavior before release?

Reply:
Honestly, I don't recall. This PR is so old, I've completely forgotten the context of this code, but I think this came from example code somewhere. I'll have to dig to find out the reason for it.